lib/model/compose.dart

```diff
@@ -175,3 +176,7 @@ Uri narrowLink(PerAccountStore store, Narrow narrow, {int? nearMessageId}) {
 
   return store.account.realmUrl.replace(fragment: fragment.toString());
 }
+
+String mention(User user, {bool silent = false}) {
+  return '@${silent ? '_' : ''}**${user.fullName}|${user.userId}**';
+}
```

Comment:
Perhaps doesn't need a reusable function defined here, but:

- Having tests for this logic can be nice
- We'll also be writing mentions for #49 / #129.